index.js

```diff
@@ -184,6 +207,12 @@ var Umzug = module.exports = redefine.Class({
     }
   },
 
+  log: function() {
+    if (this.options.logging) {
+      this.options.logging.apply(null, [].slice.call(arguments));
```

Comment:
A user of Umzug who is writing a logging method needs to know what to do with multiple arguments - e.g. concat them all with .toString?  Or is the first argument a context object, the second a template, and subsequent args params as per Bunyan logging?  Conversely the Umzug code needs to know when and how to call the log method with multiple args.  I think probably easiest would be just to have one argument that is a string - that's all you're using today AFAICT anyway.